examples/next-app-router-ssr/lib/api-graphql.ts

```diff
@@ -21,13 +28,22 @@ interface FetchResponse {
 }
 
 const POST_GRAPHQL_FIELDS = `
-__typename
-sys {
-  id
-}
-slug
-title
-description
+  __typename
+  sys {
+    id
+  }
+  slug
+  title
+  description
+  banner {
+    sys {
+      id
+    }
+    title
+    file {
```

Comment:
just double-checked here: https://www.contentful.com/developers/docs/references/graphql/
it should be directly URL without `file`